examples/layers.ipynb

```diff
@@ -251,9 +187,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python [conda env:napari-gui]",
```

Comment:
does anyone know if I can just delete this or should I reset it to "reasonable" defaults?